app/views/pages/medicaid/confirmation.erb

```diff
@@ -0,0 +1,43 @@
+<div class="step-header">
+  <a href="/pages/medicaid/contact_ss_dob" class="step-header__back-link">
+    <i class="button__icon icon-arrow_back"></i>
+  </a>
+  <h4 class="step-header__title">
+    Confirmation
+  </h4>
+</div>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="form-card__title">Thank you for submitting an application!</div>
+  </header>
+
+  <div class="form-card__confirmation">
+    <p class="text--help text--centered">
+      Date: October 4, 2017
+    </p>
+
+    <p class="text--help text--centered">
+      Applicant(s): Harriett Fletcher
+    </p>
+
+    <p class="text--help text--centered">
+      Benefits: Medicaid, medical expense coverage
```

Comment:
it seems like this text will be dynamic (based on the application), and possible values would be "Food assistance" and "Medicaid" or both -- is that right? If so, can we change this to reflect a possible value? (not sure how someone would apply for medical expense coverage)

Reply:
Yes, I'll update it to just "Medicaid"